test/simulation/actions.go

```diff
@@ -228,7 +228,7 @@ func (a *actions) placeOrderAndSwap(srcNode, destNode *xudtest.HarnessNode,
 
 	// Verify the swap events info.
 	a.assert.Equal(eMaker.swap.OrderId, eTaker.swap.OrderId)
-	a.assert.NotEqual(eMaker.swap.LocalId, eTaker.swap.LocalId)
+	//a.assert.NotEqual(eMaker.swap.LocalId, eTaker.swap.LocalId)
```

Comment:
Why commenting it our instead of removing this line entirely?

Reply:
I can remove it if you say so guys, can't imagine why would we require it @sangaman